news/_posts/2020-12-09-pr-testing-on-prestashop.md

```diff
@@ -0,0 +1,179 @@
+---
+layout: post
+title:  "Testing Pull Requests on PrestaShop"
+subtitle: "How does the QA team handle the testing of Pull Requests?"
+date:   2020-12-08 16:00:00
+authors: [ RobinFischer ]
+icon: icon-bug
+tags: [tests, community, contribute]
+---
+
+The PrestaShop company created a Quality Assurance team 4 years ago, in 2016, to improve the quality of the open source project – but also of its products and services. Over the last year, the team has grown quickly, taking quality to the next level! Its field of action has been widening (tests of new services like [Prestashop Checkout](https://www.prestashop.com/en/prestashop-checkout), specifications reviews, etc.) but the core of the manual testers' activity remains the same: verifying Pull Requests (“PRs”) on GitHub, and testing every release.
+
+
+# I. Some questions you may have in mind
+
+## Why test every Pull Request?
+
+The first question that might come to your mind: why is there a QA verification on each Pull Request? Why not test the release build only?
+
+- The earlier a bug is detected in the process, the easier it is to find the source of the problem and fix it
+![The cost of fixing a bug rises over time](/assets/images/2020/12/cost_bug_time.jpg)
+- Testing each Pull Request allows the QA analysts to perform more detailed tests, improving the overall software quality
+- There can be several months between the moment a PR is merged and the time the build is first tested, enough for the author of the PR to have “forgotten” what they did exactly.
+
+## How do you choose which Pull Request should be tested first?
+
+This is a legitimate question that pops up regularly. It might be frustrating for a contributor to see its Pull Request blocked in “Waiting for QA” for weeks, while other PRs are validated and merged the same day they are created.
```

Comment:
```suggestion
This is a legitimate question that pops up regularly. It might be frustrating for a contributor to see its Pull Request  waiting for weeks on GitHub with the “Waiting for QA” label, while other PRs are validated and merged the same day they are created.
```